app/javascript/components/pages/HomePage.jsx

```diff
@@ -1,123 +1,306 @@
+                    </div>
+
+                    <div className='shading overlay'></div>
+
+                    <div className='text'>
+                        <Typography variant="h1">
+                            {props.item.name}
+                        </Typography>
+
+                        <Typography variant="h4">
+                            {props.item.description}
+                        </Typography>
+
+                        <Button
+                            variant="contained"
+                            color="secondary"
+                            component={Link}
+                            to={String('/'+ props.item.link)}
+                            onClick={ () => {console.log(String('/'+ props.item.link), "button clicked.")} }
+                        >
+                            {props.item.button}
+                        </Button>
+                    </div>
+                </div>
+            </div>
+        </Paper>
+    )
+}
+
+const HomePage = (props) => {
+   var retData;
+  const [inputList, setInputList] = useState([{ virtual_link: "", building: "", room: "", city: "", stateloc: "", start_time: "", end_time: "" }]);
+  const getInputList = () => {
+      const url = "/api/v1/locations";
+      fetch(url)
+          .then(response => {
+              if (response.ok) {
+                 //retData = response.json();
+                  return response.json();
+              }
+              throw new Error("Network response was not ok.");
+          })
+          .then(response => {
+            console.log(response);
+            //setInputList(retData);
+            setInputList([...response, { virtual_link: "", building: "", room: "", city: "", stateloc: "", start_time: "", end_time: "" }])
+          })
+         // .then(response => setInputList( response ))
+         .catch(error => console.log(error));
+  }
+
+/*  function loadInputList() {
+    useEffect(() => {
+      getInputList()
+    }, []);
+  }*/
+
+
+// const [attendances, setAttendances] = React.useState(undefined);
+const [attendances, setAttendances] = useState([]);
+console.log("Here")
+
+// componentDidMount
+ const fetchAttendance = () => {
+    const url = "/attendances";
+    fetch(url)
+        .then(response => {
+            if (response.ok) {
+                return response.json();
+            }
+            throw new Error("Network response was not ok.");
+        })
+        .then(response => {
+            console.log(response);
+            setAttendances(response);
+
+        })
+        .catch(err => console.log("Error: " + err));
+}
+
+/* const fetchAttendance = () => {
+    const url = "/attendances";
+    const response = fetch(url);
+    const json = response.json;
+    setAttendances(json.data);
+    console.log(attendances);
+  }
+  */
+
+React.useEffect(() => {
+    fetchAttendance();
+    getInputList();
+    console.log(attendances);
+}, []);
+
+
+
+const getTimeTracker = () => {
+

```

Comment:
```suggestion
count1 = 0;
tot_time = 0;
```
Need to do this to avoid the total meditation hours from duplicating on refresh.